app/core/utils.coffee

```diff
@@ -796,7 +796,7 @@ seasons = [
 currentSeason = () ->
   now = new Date()
   year = now.getFullYear()
-  return seasons.find((season) -> now <= new Date("#{year}-#{season.end}"))
+  return seasons.find((season) -> now < new Date("#{year}-#{season.end}").setHours(24, 0, 0, 0))
```

Comment:
This still fails for me, but works like this with the test cases we have:
```suggestion
  return seasons.find((season) -> now <= new Date("#{year}-#{season.end}").setHours(24, 0, 0, 0))
```

Not sure if our tests are comprehensive enough, but does this make sense to do? When run from where I am, `now` logs as `Tue Dec 31 2013 00:00:00 GMT-0800 (Pacific Standard Time)`, so the very beginning of December 31, whereas the threshold date after the `setHours` call is also exactly `Tue Dec 31 2013 00:00:00 GMT-0800 (Pacific Standard Time)`. (It seems like these are all off-by-one and that we should be in season 3 all the way until the end of Dec 31, not just until the beginning of Dec 31?)

The other thing we could do to make this more timezone aware is to note that seasons end at 23:59:59 in Pacific time (which is PDT, UTC-8, for seasons 1 and 2 and PST, UTC-7, for season 3).